ParkenDD/SettingsView/SettingsViewController.swift

```diff
@@ -253,6 +254,14 @@ class SettingsViewController: UITableViewController, MFMailComposeViewController
 					let tweetsheet = SLComposeViewController(forServiceType: SLServiceTypeTwitter)
 					tweetsheet?.setInitialText(L10n.tweetText.string)
 					self.present(tweetsheet!, animated: true, completion: nil)
+				} else {
+					let stringURL = "https://twitter.com/"
+					guard let url = URL(string: stringURL) else {
```

Comment:
Should use a `!` here to make the code cleaner. If URL cannot make a URL out of "https://twitter.com" Apple has a big problem.